Domain/Cell/State/EmptyCellState.cs

```diff
@@ -0,0 +1,24 @@
+namespace DPAT_eindopdracht.Domain.Cell.State;
+
+public class EmptyCellState : BaseCellState
+{
+    public EmptyCellState(Cell context) : base(context)
+    {
+    }
+
+    public override void SetFixedValue(int? value)
+    {
+        if (value > 0)
+        {
+            base.SetFixedValue(value);
+        }
```

Comment:
Je checkt vaker of de value > 0 is. Allereerst wat doen we als dit niet zo is. En is er ook een regel wat het hoogste getal kan zijn. Wat mij betreft nu 9 namelijk. Al kan dat altijd anders. Maar dan zou -1 ook gebruikt kunnen gaan worden